pom.xml

```diff
@@ -35,7 +35,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <ddf.version>2.10.0</ddf.version>
+        <ddf.version>2.11.0-SNAPSHOT</ddf.version>
```

Comment:
Just talked with Shaun and they are looking at cutting a release for DDF 2.10.2 today.